doc/usecases/current/membership_application.md

```diff
@@ -0,0 +1,42 @@
+# Use Case: Membership Application
+
+## Summary
+
+- **Scope:** Membership
+- **Level:** User Goal
+- **Actors:** User (Membership Applicant), Administrator (Chairman)
+- **Brief:** The user can apply for a membership (account)
+- **Assignee:** Thorben (Frontend), Lukas (Keycloak Installation)
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and is not logged in.
+- **Main success scenario:**
+  1. The user can click an register button which leads them to Keycloak to fill out:
+     - username
+     - first+last name
+     - email address
+  2. The user can look at public maps (read-only and without seeing addresses!)
+  3. The user can visit a 3-step form to apply for membership, with:
+     - user photo (for Nextcloud)
+     - telephone number (optional)
+     - website
```

Comment:
I think this as well as organsiation should be optional, since not every person could fill this out. Unless of course you only want a very small and specific fraction of people to be members.

Reply:
Yes, I fully agree, fixed in 5c60792957f1cbd79518fbd4c973ac78e9a0cdfa! I also made it more consistent to Yvonne's forms (with a few additional ones). Some items were moved to the 3rd form. (1st form got crowded). Form 2 is better explained in Yvonne's picture, I didn't write all the details in the use case.